templates/github/.ci/ansible/Containerfile.j2.copy

```diff
@@ -17,7 +19,7 @@ RUN pip3 install
 {{ " " }}{{ item.source }}
 {%- if item.name == "pulpcore" -%}
 {%- if s3_test | default(false) -%}
-[s3]
+[s3] git+https://github.com/fabricio-aguiar/botocore.git@fix-100-continue
```

Comment:
I wonder if we needed a rebased version of this fix. Also how do we handle this in production?
Do we need to vendor botocore?

Reply:
It only affects MinIO, so if users are using that as their backend they will run into a timeout issue after a 0-byte file is uploaded. Luckily with Pulp's artifact system this will only occur once as the second time a 0-byte file is downloaded/uploaded the artifact will be re-used. Also, looking at the Deb tests that trigger this timeout it doesn't actually fail the sync since we have retries on downloads after the timeout. It would probably be good to use a rebased version of this fix.

Also, I've come to realize this bit of code for backend specific installs only works in the pulpcore repository since the `plugins` list will usually only have the plugin it's testing.

Second thing I've realized is that our lowerbounds test runner is no longer doing anything with the addition of the build step. We modify the `requirements.txt` after the build step has already ran. We probably need to use the output of `.ci/scripts/calc_deps_lowerbounds.py` as a constraint file for the pip install. I'll see if I can modify this PR to include that.

Reply:
I think, i get what you say. We only pass the built whl to pip inside the container forgetting about the just adapted requirements.txt. I'll find a solution for that.